tests/test_actions.py

```diff
@@ -126,6 +127,8 @@ def test_update_request_with_external_calls(client):
     with mock.patch("requestor.request_utils.requests") as mock_requests:
         # update the request status
         res = client.put(f"/request/{request_id}", json={"status": "APPROVED"})
+        assert res.status_code == 200, res.text
```

Comment:
Tests Fail early now. Good 👍

Reply:
it makes more sense to check the status code before running `assert_called_once_with` and getting a weird error when the request failed